1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,38 @@
-function setAlarm() {}
+// let divEl = document.querySelector(".centre");
+//   let h1Element = document.getElementById("timeRemaining");
+//  let inputVal = document.getElementById("alarmSet").value;
+
+//  let setAlarmButton = document.getElementById("set");
+// function setAlarm() {
+//   h1Element.innerText = `time Remaining:${inputVal}`
+// } 
+//   console.log(divEl);
+//    console.log(h1Element)
+// console.log(inputEl);
+// setAlarmButton.addEventListener("click", setAlarm)
+
+function setAlarm() {
+  let inputValue = document.querySelector("#alarmSet");
+  let remainder = document.querySelector("#timeRemaining");
+  let countDown = inputValue.value;
+
+  let timer = setInterval(() => {
+    const min = Math.floor(countDown / 60);
```

Comment:
Lets make: "60" a constant as we use it multiple times: `const SECONDS_IN_A_MINUTE=60`